linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/smartcard_configure_cert_checking/oval/shared.xml

```diff
@@ -1,34 +1,26 @@
 <def-group>
   <definition class="compliance" id="smartcard_configure_cert_checking" version="4">
-    <metadata>
-      <title>Enable Smart Card Login</title>
-      <affected family="unix">
-        <platform>multi_platform_sle</platform>
```

Comment:
I think that if you enable this on all platforms you should remove the rhel8 from the prodtype in the rule becasue on rhel8 neither the configuration file nor the package exist.

Reply:
So, rhel8 is in the prodtype but the rule is not in any rhel8 profile. That's probably a leftover?
If that's right, I can remove rhel8 from the prodtype in a new commit.

Reply:
I've removes rhel8 from prodtype in a new commit.